### PR TITLE
[minor] SCA

### PR DESCRIPTION
**Pull request by kalessil**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

New findings: language level, greedy regex, array_column usages

### CODE REVIEW

**Comment by kalessil** [comment]
array_column doesn't preserve original keys, hence feedback is appreciated.

File: src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ private function connect()
     private function disconnect()
     {
         if ($this->connection && is_resource($this->connection)) {
-            ldap_close($this->connection);
+            ldap_unbind($this->connection);
```
Comment (javiereguiluz) [inline src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php]: This is the more confusing change ... but if you look up this function: http://php.net/ldap_close you see that `ldap_close` is just an alias of `ldap_unbind`.

**Review by javiereguiluz — APPROVED** [review]
@kalessil I love the `array_column()` simplification. Thanks!

**Review by derrabus — APPROVED** [review]

**Review by 20uf — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by kalessil** [comment]
Thank you @javiereguiluz =)

File: src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php (hunk 1 above)
Comment (kalessil) [reply src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php]: Indeed. Some aliases are also not documented, for my inspection lots of aliases had to be extracted from the source code :)

**Comment by nicolas-grekas** [comment]
Thank you @kalessil.

**Pull request merged**